### PR TITLE
[zh-cn]: update translation of `<slot>` element

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/html/element/slot/index.md [hunk 1]
```diff
@@ -81,21 +51,82 @@ slug: Web/HTML/Element/slot
...
+      <td>{{domxref("HTMLSlotElement/slotchange_event", "slotchange")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">标签省略</th>
+      <td>不允许，开始标签和结束标签都不能省略。</td>
+    </tr>
+    <tr>
+      <th scope="row">允许的父元素</th>
+      <td>
+        任何接受<a href="/zh-CN/docs/Web/HTML/Content_categories短语内容"
```
Comment (T34-active) [inline files/zh-cn/web/html/element/slot/index.md:107]:
```suggestion
        任何接受<a href="/zh-CN/docs/Web/HTML/Content_categories#短语内容"
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 2]
```diff
@@ -1,53 +1,23 @@
 ---
-title: <slot>
+title: <slot>：Web Component 插槽元素
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:2]:
```suggestion
title: <slot>：Web 组件插槽元素
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 3]
```diff
@@ -1,53 +1,23 @@
...
-   <th scope="row">Permitted ARIA roles</th>
-   <td>None</td>
-  </tr>
-  <tr>
-   <th scope="row">DOM interface</th>
-   <td>{{domxref("HTMLSlotElement")}}</td>
-  </tr>
- </tbody>
-</table>
+**`<slot>`** [HTML](/zh-CN/docs/Web/HTML) 元素是——[Web Components](/zh-CN/docs/Web/API/Web_components) 技术套件的一部分——它是一个在 web component 内部的占位符，你可以使用自己的标记来填充该占位符，从而创建单独的 DOM 树并将其一起呈现。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:10]:
```suggestion
**`<slot>`** [HTML](/zh-CN/docs/Web/HTML) 元素是——[Web 组件](/zh-CN/docs/Web/API/Web_components)技术套件的一部分——它是一个在 web 组件内部的占位符，你可以使用自己的标记来填充该占位符，从而创建单独的 DOM 树并将其一起呈现。
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 4]
```diff
@@ -1,53 +1,23 @@
...
 
-此元素有[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
+此元素仅包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
 
 - `name`
-  - : 插槽的名字。拥有 name 属性的插槽叫**具名插槽**。
+
+  - : 插槽名称。
+
+    **_命名插槽_** 是具有 `name` 属性的 `<slot>` 元素。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:20]:
```suggestion
    **_具名插槽_**是具有 `name` 属性的 `<slot>` 元素。
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 5]
```diff
@@ -81,21 +51,82 @@ slug: Web/HTML/Element/slot
   </style>
   <details>
     <summary>
-      <code class="name"
-        >&lt;<slot name="element-name">NEED NAME</slot>&gt;</code
-      >
-      <i class="desc"><slot name="description">NEED DESCRIPTION</slot></i>
+      <code class="name">
+        &lt;<slot name="element-name">需要的名称</slot>&gt;
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:55]: _Need_, not _needed_

```suggestion
        &lt;<slot name="element-name">需要名称</slot>&gt;
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 6]
```diff
@@ -81,21 +51,82 @@ slug: Web/HTML/Element/slot
...
+      </code>
+      <span class="desc"><slot name="description">需要的描述</slot></span>
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:57]:
```suggestion
      <span class="desc"><slot name="description">需要描述</slot></span>
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 7]
```diff
@@ -81,21 +51,82 @@ slug: Web/HTML/Element/slot
...
+      <h4>属性</h4>
+      <slot name="attributes"><p>无</p></slot>
     </div>
   </details>
   <hr />
 </template>
 ```
 
-> **备注：** 你可以在 [element-details](https://github.com/mdn/web-components-examples/tree/master/element-details) (see it [running live](https://mdn.github.io/web-components-examples/element-details/)) 找到这个示例的完整演示版。另外，你也可以在[Using templates and slots](/zh-CN/docs/Web/API/Web_components/Using_templates_and_slots)这里找到具体的解释。
+> **备注：** 你可以在 [element-details](https://github.com/mdn/web-components-examples/tree/main/element-details)（参见[在线演示](https://mdn.github.io/web-components-examples/element-details/)）查看此完整示例。此外，你还可以在[使用模板和插槽](/zh-CN/docs/Web/API/Web_components/Using_templates_and_slots)找到相关解释。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:68]:
```suggestion
> **备注：** 你可以在 [element-details](https://github.com/mdn/web-components-examples/tree/main/element-details)（参见[在线演示](https://mdn.github.io/web-components-examples/element-details/)）查看完整示例。此外，你还可以在[使用模板和插槽](/zh-CN/docs/Web/API/Web_components/Using_templates_and_slots)找到相关解释。
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 8]
```diff
@@ -81,21 +51,82 @@ slug: Web/HTML/Element/slot
...
+        >、<a href="/zh-CN/docs/Web/HTML/Content_categories#短语内容"
+          >短语内容</a
+        >。
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">允许的内容</th>
+      <td>
+        <a href="/zh-CN/docs/Web/HTML/Content_categories#透明内容模型"
+          >透明内容模型</a
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:92]:
```suggestion
          >透明内容</a
```

File: files/zh-cn/web/html/element/slot/index.md [hunk 9]
```diff
@@ -104,3 +135,10 @@ slug: Web/HTML/Element/slot
 ## 浏览器兼容性
 
 {{Compat}}
+
+## 参见
+
+- HTML {{HTMLElement("template")}} 元素
+- HTML [`slot`](/zh-CN/docs/Web/HTML/Global_attributes/slot) 属性
+- CSS {{CSSXref("::slotted")}} 伪元素
+- [CSS scoping](/zh-CN/docs/Web/CSS/CSS_scoping) 模块
```
Comment (yin1999) [inline files/zh-cn/web/html/element/slot/index.md:144]: https://www.w3.org/2019/11/web-roadmaps/mobile/adaptation.zh.html

```suggestion
- [CSS 域](/zh-CN/docs/Web/CSS/CSS_scoping)模块
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Review by yin1999 — APPROVED** [review]
👍

**Pull request merged**